src/components/admin/form-components/validated-fields.jsx

```diff
@@ -24,6 +24,14 @@ export const shorterThanDbVarChar = value => {
   return null;
 };
 
+export const isPrettySlug = value => {
+  const prettySlug = /^[a-z-]+$/g;
+  if (value && !prettySlug.test(value)) {
```

Comment:
Also, did we also want to check it was max 50 characters or something like that?

Reply:
I thought about that and kinda of the opinion that should be the editors call. Preventing weird characters like ", ', spaces, :, etc. falls under our responsibility since those things have the potential to cause tech problems, but I feel like the length of the Slug should be an editorial decision since it's mostly stylistic.